docs/components/radios/index.html

```diff
@@ -4,6 +4,7 @@
 also:
   components/radios/test.html: Testing - Radios
   elements/hx-radio-control: <hx-radio-control>
+  elements/hx-radio-set: <hx-radio-set>
```

Comment:
please alphabetize (nothing before something)

Reply:
same as above comment, i believe we're doing something before nothing, because originally in this file elements/hx-radio-control: <hx-radio-control> was above elements/hx-radio <hx-radio>  @CITguy thoughts?